lxd/cluster/member_state.go

```diff
@@ -73,6 +74,10 @@ func LocalSysInfo() (*api.ClusterMemberSysInfo, error) {
 		return nil, fmt.Errorf("Failed getting load averages: %w", err)
 	}
 
+	// NumCPU gives the number of threads available to the LXD server at startup,
+	// not the currently available number of threads.
+	sysInfo.CPUThreads = uint64(runtime.NumCPU())
```

Comment:
Any particular reason for using "Threads" here? Was it to align with a similar field elsewhere?

The NumCPU function says "NumCPU returns the number of logical CPUs usable by the current process.", so I wondered if "LogicalCPUs" would be clearer?